.storybook/stories/ResetButton.stories.tsx

```diff
@@ -0,0 +1,18 @@
+import type { ComponentMeta, ComponentStory } from "@storybook/react";
+
```

Comment:
Seems that Prettier or VSCode added a line between the imports. I might have to check if the auto-formatting is working as expected u.u